src/App.jsx

```diff
@@ -14,57 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const clearClick = () => {
```

Comment:
`clearClick` is not the best naming for this function: sounds like it's clearing the click handler. The more proper name would be `clearSelectedGood` or `onClearButtonClick`, so that it explicitly describes this function's purpose.